libkpod/containerserver.go

```diff
@@ -113,7 +120,147 @@ func New(config *Config) (*ContainerServer, error) {
 			sandboxes:  make(map[string]*sandbox.Sandbox),
 		},
 		config: config,
-	}, nil
+	}
+
+	return c, c.Update()
+}
+
+// Update makes changes to the server's state (lists of pods and containers) to
+// reflect the list of pods and containers that are stored on disk, possibly
+// having been modified by other parties
+func (c *ContainerServer) Update() error {
```

Comment:
It looks like in order to do this, I also have to move the storageRuntimeServer over to ContainerServer so that the ContainerServer can determine if it is looking at a pod or a container